### PR TITLE
Enable LTP test case on WSL

### PR DESCRIPTION
**Pull request by squirrelsc**
See details in commits.

### CODE REVIEW

**Comment by LiliDeng** [comment]
@squirrelsc please rebase and resolve the conflict

**Comment by squirrelsc** [comment]
> @squirrelsc please rebase and resolve the conflict

Sure. I found shell doesn't supported WSL. I will add more commits.

**Comment by squirrelsc** [comment]
@LiliDeng Ready to merge

File: lisa/tools/lsblk.py [hunk 1]
```diff
@@ -155,57 +151,37 @@ def get_disks(self, force_run: bool = False) -> List[DiskInfo]:
         disks: List[DiskInfo] = []
 
         # parse output of lsblk
+        # -b print SIZE in bytes rather than in human readable format
+        # -J output in JSON format
+        # -o list of columns to output
         output = self.run(
-            "-b -P -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
+            "-b -J -o NAME,SIZE,TYPE,MOUNTPOINTS,FSTYPE", sudo=True, force_run=force_run
```
Comment (LiliDeng) [inline lisa/tools/lsblk.py]: MOUNTPOINTS doesn't work on linux. It is `MOUNTPOINT`

Comment (squirrelsc) [reply lisa/tools/lsblk.py]: I tested it with Ubuntu 22.04 WSL. Which distro doesn't work? The MOUNTPOINT returns just one mount point. In WSL, it doesn't return the root path `/` by default.

Comment (LiliDeng) [reply lisa/tools/lsblk.py]: I tested it on marketplace image `Canonical 0001-com-ubuntu-server-focal 20_04-lts latest`

Comment (squirrelsc) [reply lisa/tools/lsblk.py]: ok, I will change it back to MOUNTPOINT for compatibility. It should also work.

**Review by LiliDeng — APPROVED** [review]

File: lisa/tools/lsblk.py [hunk 2]
```diff
@@ -155,57 +151,42 @@ def get_disks(self, force_run: bool = False) -> List[DiskInfo]:
         disks: List[DiskInfo] = []
 
         # parse output of lsblk
+        # -b print SIZE in bytes rather than in human readable format
+        # -J output in JSON format
+        # -o list of columns to output
         output = self.run(
-            "-b -P -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
+            "-b -J -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
```
Comment (LiliDeng) [inline lisa/tools/lsblk.py:158]: there is a regression for this change.
On some images like `openlogic centos 7_9-arm64 7.9.2023010302`, lsblk doesn't support -J parameter, will fix it tomorrow.

Comment (squirrelsc) [reply lisa/tools/lsblk.py:158]: Thank you, Lili! We can use -J by default, if it doesn't work, use previous table format.

**Pull request merged**